website/docs/docs/collaborate/navigate-query-page.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Navigate the Query page interface"
+description: "Learn how to navigate the Query page interface"
+sidebar_label: "Navigation interface"
+tags: [Query page]
+---
+
+# Navigate the Query page interface <Lifecycle status="beta" />
+
+<IntroText>
+Learn how to navigate the Query page interface and use the main components.
+</IntroText>
+
+:::tip
+Available in private beta. To join, please reach out to your account manager. To provide feedback, use this feedback form…
+:::
+
+The Query page provides an interactive interface for writing, running, and analyzing SQL queries. This section highlights the main components of the Query page. 
+
+### Query console
+- The Query editor is the main component of the Query page. It allows you to write, run, and analyze SQL queries
+- The Query editor supports syntax highlighting, code completion, and linting.
+- The Query editor also supports asset linking, which allows you to link to specific tables, columns, and metrics in your project.
```

Comment:
I don't know what this means! let's remove

Reply:
hyperlinking basically! removed!